prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,15 @@
+
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres']
+console.log(languages)
+
+let firstElement = languages[0]
+console.log("The firt element in the array is " + firstElement)
+
+let thirdElement = languages[2]
+console.log("The third element in the array is " + thirdElement)
+
+let length = languages.length
+console.log("The length of the array is " + length)
+
+let lastElement = languages[5]
+console.log("The last element in the array is " + lastElement)
```

Comment:
Here you can subtract `1` from your `length` variable. That way we don't have to know the actual length of the array in advance.

```js
var lastElement = languages[length - 1];
```